solutions/ModernSearch/react-search-refiners/spfx/src/webparts/searchRefiners/components/Templates/DateRange/DateRangeTemplate.tsx

```diff
@@ -77,6 +77,11 @@ export default class DateRangeTemplate extends React.Component<IDateRangeTemplat
         }
 
         return <div>
+            {
+                this.props.showValueFilter ? 
+                    <div className='ms-font-s'>Value filters are not allowed for dates. Clear 'show filter' to remove this message</div>
```

Comment:
@FranckyC do you know if these will "just work" meaning do we have "ms-Fabric" as a top class here? Or is it better to wrap in an internal css class?

Reply:
This should work bu it uses the global ms-Fabric class.